azure/aks-full-sample/pom.xml

```diff
@@ -4,36 +4,32 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.yanchware.fractal</groupId>
-    <artifactId>aks.full.sample</artifactId>
-    <version>1.0-SNAPSHOT</version>
-
+    <parent>
+        <groupId>com.yanchware.fractal</groupId>
+        <artifactId>azure</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
Maybe we should remove SNAPSHOT?

Reply:
yeah, and maybe use semantic versioning.
we could put that it is version `1.0.0` or `1.1.0`